bundles/mapping/mapmodule/plugin/publishertoolbar/PublisherToolbarPlugin.js

```diff
@@ -25,50 +25,15 @@ Oskari.clazz.define('Oskari.mapframework.bundle.mapmodule.plugin.PublisherToolba
         me._defaultLocation = 'top left';
         me._index = 0;
         me._name = 'PublisherToolbarPlugin';
-
-        me.toolbarId = 'PublisherToolbar';
-        me.toolbarContent = 'publishedToolbarContent';
-        me.toolbarPopupContent = 'publishedToolbarPopupContent';
-        me.toolbarContainer = 'publishedToolbarContainer'; // Note! this needs to match styles and templates
-
-        me._mobileDefs = {
-            buttons: {
-                'mobile-publishedtoolbar': {
-                    iconCls: 'mobile-menu',
-                    tooltip: '',
-                    sticky: true,
-                    toggleChangeIcon: true,
-                    show: true,
-                    callback: function () {
-                        if (me.popup && me.popup.isVisible()) {
-                            me._closeToolsPopup();
-                            var sandbox = me.getSandbox();
-                            var toolbarRequest = Oskari.requestBuilder('Toolbar.SelectToolButtonRequest')(null, 'mobileToolbar-mobile-toolbar');
-                            sandbox.request(me, toolbarRequest);
-                        } else {
-                            me._openToolsPopup();
-                        }
-                    }
-                }
-            },
-            buttonGroup: 'mobile-toolbar'
-        };
-
-        me._buttons = conf.buttons || [];
+        me._toolButtons = [];
```

Comment:
We'll probably need to initialize toolButtons like the buttons array was before so we can restore the selections on embedded map. Otherwise this only works on the publisher functionality itself?